### ISSUE TITLE
Deno websocket server with two clients crashes with broken pipe

### ISSUE THREAD

**Issue by dionjwa** [denoland/deno#19498]
Deno 1.32.5 (docker image `denoland/deno:alpine-1.32.5`)

I don't have an exact reproduction yet, but it's a simple websocket server, with two clients sending JSON over websockets. 

The error doesn't show anywhere in my code where I am e.g. missing a try/catch.

It usually happens around connection/disconnection.

The error is too vague for me to take any action.

```
| error: Uncaught (in promise) TypeError: Broken pipe (os error 32)
|     at eventLoopTick (ext:core/01_core.js:165:11)
```

**Comment by bartlomieju** [comment]
Can you reproduce it on the latest Deno version?

**Comment by sust4in** [comment]
> Can you reproduce it on the latest Deno version?

yes, we also have similar issue on same scenario.

error: Uncaught TypeError: Broken pipe (os error 32)
    at async ext:deno_websocket/01_websocket.js:509:15

on 1.34.2

**Comment by jespertheend** [comment]
@mmastrac I'm still seeing this on Deno 1.38.0. I'll try to get a reduced test case going.

**Comment by jespertheend** [comment]
Turns out I've been making a reduced test case for a separate issue (#21121). I was so focused on some of my messages not coming through that I didn't realise the broken pipe error stopped being thrown.
I'll wait with making another reduced test case since the two are possibly related.

**Comment by mmastrac** [comment]
Might be related to https://github.com/denoland/deno/issues/19707 as well

**Comment by mmastrac** [comment]
I believe this will be fixed with #21205

**Comment by jespertheend** [comment]
I can confirm that that PR fixes both my broken pipe crash as well as #21121.

**Comment by jespertheend** [comment]
This is great! I think I can finally start upgrading from Deno 1.32.2 🥲
